newrelic/hooks/logger_structlog.py

```diff
@@ -64,6 +68,8 @@ def new_relic_event_consumer(logger, level, event):
                 event = message
             elif isinstance(event, dict) and "event" in event:
                 event["event"] = message
+            elif isinstance(event, tuple) and event[0][0] and "event" in event[0][0]:
+                event[0][0]["event"] = message
```

Comment:
Let's throw this into a try/except block to prevent crashes, make this an else and then an if underneath.